packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -38,7 +38,17 @@ const LineControls = forwardRef(function LineControls({
   `
 
   function onEnterOrSpace(ev, func) {
-    if (ev.keyCode === 13 || ev.keyCode === 32) func()
+    if (ev.keyCode === 13 || ev.keyCode === 32) {
```

Comment:
We could use `event.key` here, which is used elsewhere in the classifier and is easier to read than key codes.
```suggestion
    if (ev.key === 'Enter' || ev.key === ' ') {
```